server.h

```diff
@@ -33,10 +32,13 @@ struct cg_server {
 	struct cg_output *output;
 	struct wl_listener new_output;
 
+	struct wl_listener xdg_toplevel_decoration;
+	struct wl_listener new_xdg_shell_surface;
 #if CAGE_HAS_XWAYLAND
 	struct wl_listener new_xwayland_surface;
 #endif
 
+	bool xdg_decoration;
```

Comment:
Does this not need to be assigned a default value?

Reply:
[It is](https://github.com/Hjdskes/cage/blob/master/cage.c#L121).